docs/content/integrations/dbt/quickstart.mdx

```diff
@@ -0,0 +1,264 @@
+---
+title: "Quickstart: Run your dbt project with Dagster"
+description: Get started swiftly with this simple dbt + Dagster example.
```

Comment:
nit: no `+` since that's might be confused with `Dagster+`

```suggestion
description: Get started swiftly with this simple dbt & Dagster example.
```